recipes/powershell4.rb

```diff
@@ -21,6 +21,8 @@
 # PowerShell 4.0 Download Page
 # http://www.microsoft.com/en-us/download/details.aspx?id=40855
```

Comment:
See the comment below about `\\powershell\\3` in the registry (not part of this commit) -- shouldn't we be checking for `4`?